.env.example

```diff
@@ -26,3 +26,9 @@ CHES_BASEURL=
 CHES_SSO_TOKEN_URL=
 CHES_SSO_CLIENT_ID=
 CHES_SSO_CLIENT_SECRET=
+
+# Microsoft Azure Application Details
+AZURE_CLIENT_ID=
+#May be called "password" rather than "secret" in the authoriaztion details
+AZURE_CLIENT_SECRET=
```

Comment:
I'm see this addition to the `.env.example`, but I don't see these variables being used, except when they're passed through the `docker-compose` file. I think it's a good idea to leave these here and use them in the React front-end.